src/components/avatar/utils.ts

```diff
@@ -63,3 +78,39 @@ export function rgbToHue(rgb: RGB): number {
 export function hexToHue(hex: string): number {
   return rgbToHue(hexToRGB(hex));
 }
+
+/**
+ * For RGB color, find luminance
+ *
+ * @param colorObject {RGB}
+ */
+export function rgbToLuminosity(colorObject: RGB): number {
+  const color = Color(colorObject);
+  const luminosity = color.luminosity();
+  return luminosity;
+}
+
+/**
+ * Verify the generated color is WCAG contrast compliant
+ *
+ * @param hex {string} - form of "#------"
+ * @param theme {string}
+ */
+export function isContrastCompliant(hex: string, theme: string): string | null {
+  const getLuminance = (hex) => {
+    const rgbObj = hexToRGB(hex);
+    return rgbToLuminosity(rgbObj);
+  };
+  const themeInitials = theme === "light" ? "#6A6A6A" : "#9f9f9f";
+
+  //dark mode uses ui-text-3-dark #9f9f9f (lighter grey initials, darker background)  luminosity 0.35
+  //light mode uses ui-text-3 #6a6a6a (darker grey initials, lighter background) luminosity 0.144
+  const k = 0.05; //small offset to prevent division by zero
+  const [l1, l2] =
```

Comment:
`color` already provides this util: https://github.com/Qix-/color/blob/master/index.js#L276